File: atividade haskell/calculadora.hs
main :: IO ()
main = do
    putStrLn "Digite o primeiro número:"
    input1 <- getLine
    let x = read input1 :: Double

    putStrLn "Digite o segundo número:"
    input2 <- getLine
    let y = read input2 :: Double

    let subtracao = x - y
    let soma = x + y
    let multiplicacao = x * y

    if y /= 0
        then do
            let divisao = x / y
            putStrLn $ "Resultado Adição: " ++ show soma
            putStrLn $ "Resultado Subtração: " ++ show subtracao
            putStrLn $ "Resultado Multiplicação: " ++ show multiplicacao
            putStrLn $ "Resultado Divisão: " ++ show divisao
        else
            putStrLn "Não é possível dividir o número por zero."

File: atividade c/calculadora.c
#include <stdio.h>

int main() {
    double num1, num2;

    printf("Digite o primeiro número: ");
    scanf("%lf", &num1);

    printf("Digite o segundo número: ");
    scanf("%lf", &num2);

    printf("Soma: %.2lf\n", num1 + num2);
    printf("Subtração: %.2lf\n", num1 - num2);
    printf("Multiplicação: %.2lf\n", num1 * num2);

    if (num2 != 0) {
        printf("Divisão: %.2lf\n", num1 / num2);
    } else {
        printf("Erro. Divisão por 0!\n");
    }

    return 0;
}